go/vt/vtgate/planbuilder/operators/fk_cascade.go

```diff
@@ -61,9 +61,6 @@ func (fkc *FkCascade) Inputs() []Operator {
 
 // SetInputs implements the Operator interface
 func (fkc *FkCascade) SetInputs(operators []Operator) {
-	if len(operators) < 2 {
-		panic("incorrect count of inputs for FkCascade")
-	}
```

Comment:
Maybe we should put all these checks behind a debug mode? 🤔 Would that help us in debugging when a user reports a query is not working as intended? Also, won't removing these make some plans pass that would otherwise panic that could return wrong rows?

Reply:
I changed the code so that we now only use `SetInputs()` on the output of `GetInputs()`. Is that enough you think?